must-gather/collection-scripts/gather_ceph_resources

```diff
@@ -205,14 +205,20 @@ for ns in $namespaces; do
 
     oc delete -f pod_helper.yaml
 
+    crash_collection(){
+        oc debug nodes/"${node}" --to-namespace="${ns}" -- bash -c "sleep 5m" &
+        sleep 60
```

Comment:
Instead of sleeping, how about a simple loop that will check every few seconds if the pod is ready? Then as soon as it is ready we can continue. Again, it's important to avoid long sleeps.

Reply:
yes that can be done.

Reply:
done